editor/editor_properties.cpp

```diff
@@ -1802,8 +1802,13 @@ void EditorPropertyNodePath::_node_selected(const NodePath &p_path) {
 
 	NodePath path = p_path;
 	Node *base_node = Object::cast_to<Node>(get_edited_object());
-	if (base_node == NULL && get_edited_object()->has_method("get_root_path")) {
-		base_node = get_edited_object()->call("get_root_path");
+	if (base_node == NULL){
+		if (Object::cast_to<Resource>(get_edited_object())){
```

Comment:
You should have a space before the opening brackets: https://travis-ci.org/godotengine/godot/jobs/446867553#L534

Reply:
fixed.